tinywallet/tinywallet/screens.py

```diff
@@ -996,17 +1072,118 @@ def __init__(self):
         self.layout.addWidget(wgt)
 
         header = Q.makeLabel("Select an account", 20)
-        header.setContentsMargins(0, 15, 0, 5)
+        header.setContentsMargins(0, 5, 0, 5)
         header.setAlignment(Q.ALIGN_LEFT)
-        self.layout.addWidget(header)
+
+        # Indicators for the connection status of dcrdata and dcrd.
+        self.dcrdLight = Q.makeLabel("⚫", 20, color="orange")
+        lbl = Q.makeLabel("dcrd", 14)
+        dcrdBox, lyt = Q.makeRow(self.dcrdLight, lbl)
+        lyt.setSpacing(2)
+        Q.addClickHandler(dcrdBox, self.stackDcrd)
+        dcrdBox.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
+        self.dcrdataLight = Q.makeLabel("⚫", 20, color="#76d385")
```

Comment:
Maybe it's not a real problem anymore but I'd like to avoid directly using non-ASCII Unicode in source code. Can we use the Unicode name instead?